README.md

```diff
@@ -1,28 +1,157 @@
 # Jaeger OpenShift Templates
 
-Support for deploying Jaeger into OpenShift.
+## Development setup
 
-## Install OpenShift
-If you are new to OpenShift these links help you to get it running:
-* https://install.openshift.com/
-* https://github.com/minishift/minishift
-
-## All-in-one template
-Template with in-memory storage with a limited functionality for local testing and development. 
+This template uses an in-memory storage with a limited functionality for local testing and development.
 Do not use this template in production environments.
 
 To directly install everything you need:
 ```bash
-oc process -f https://raw.githubusercontent.com/jaegertracing/jaeger-openshift/<version>/all-in-one/jaeger-all-in-one-template.yml | oc create -f -
-oc delete all,template -l jaeger-infra    # to remove everything
+oc new-project jaeger-infra
+oc process -f https://raw.githubusercontent.com/jaegertracing/jaeger-openshift/master/all-in-one/jaeger-all-in-one-template.yml | oc create -f -
 ```
 
-## Troubleshooting
-Tracer sometimes fails to resolve Jaeger's address. In such a case run:
+Once everything is ready, the Jaeger Query UI can be accessed via https://jaeger-query-jaeger.127.0.0.1.nip.io/ .
+
+## Production setup
+
+### TL;DR
```

Comment:
It would be good to write a few high level words about the template, e.g. it's meant for OC on premise, uses C* agent is deployed per node and therefore requires admin rights.